govyn/endpoint.py

```diff
@@ -67,10 +68,16 @@ async def json_body_parser(req: Request, args: Dict[str, ArgDef]) -> Dict[str, A
 
 	name = list(args)[0]
 	arg_def = args[name]
+
+	def isoformat(t: str, conv_func: Callable[[str], Union[datetime, date]], d: Optional[Any]) -> Union[datetime, date]:
```

Comment:
I'm not sure if Python is smart enough to _not_ allocate this as a closure whilst nested inside another method. Because it doesn't close over any outer vars, perhaps worth moving to a top-level func?

Reply:
From what I've just read it *shouldn't* form a closure as it doesn't reference any free variables but have de-nested just in case